src/commands/migrations/generate/README.md

```diff
@@ -0,0 +1,166 @@
+# Migrations Generate Command
+
+The `storyblok migrations generate` command allows you to generate migration files for specific component fields. This is useful when you need to transform or update field values across your Storyblok content.
+
+## Basic Usage
+
+```bash
+# Generate a migration for a specific component field
+storyblok migrations generate my-component --field my-field --space 295017
+
+# Generate a migration with a custom path
+storyblok migrations generate my-component --field my-field --space 295017 --path ./custom-path
+
+# Generate a migration with a custom suffix
+storyblok migrations generate my-component --field my-field --space 295017 --suffix staging
+```
+
+## Command Options
+
+| Option | Alias | Description |
+|--------|-------|-------------|
+| `--space <spaceId>` | `-s` | (Required) The ID of the space to generate migrations for |
+| `--field <fieldName>` | `--fi` | (Required) The name of the field to migrate |
+| `--path <path>` | `-p` | Custom path to store the migration files (default: ".storyblok/migrations/{spaceId}") |
+| `--suffix <suffix>` | `--su` | Suffix to add to the file name (e.g. {component-name}-{field}.<suffix>.js) |
+| `--verbose` | `-v` | Show detailed logs and error messages |
+
+## Output Structure
+
+The command will create a migration file in the following structure:
+
+```markdown
+.storyblok/
+└── migrations/
+    └── {spaceId}/
+        └── {component-name}-{field}.js
```

Comment:
@alvarosabu same here... I feel the Readme might be outdated, as the tests and src are designed per-component, while I see here and there in the Readme things are per-field